CompetitionTemplate/src/competition/subsystems/arm/ArmSubsystem.java

```diff
@@ -53,7 +60,11 @@ public boolean isArmAtMaximumHeight() {
     }
     
     public double getArmAngle() {
-        return encoder.getDistance();
+        return encoder.getDistance() * armEncoderDistancePerPulse.get() - armEncoderCalibrationHeight.get();
```

Comment:
Distance per pulse requires setting a variable on the encoder object itself every tick. I just think it's a little cleaner to have this one liner instead of

encoder.setDistancePerPulse(armEncoderDistancePerPulse.get());
return encoder.getDistance() - armEncoderCalibrationHeight.get();

Eventually we should just make every encoder automatically have a property object created at constructor time, which would bypass this entirely.